pkg/storage/mvcc.go

```diff
@@ -5332,6 +5332,144 @@ func CanGCEntireRange(
 	return coveredByRangeTombstones, nil
 }
 
+var emptyMetaToken enginepb.MVCCMetadata
```

Comment:
nit: why not just pass `&enginepb.MVCCMetadata{}`? Actually, since `updateStatsOnGC` only cares about `meta != nil`, it would be even better to pass a boolean with firmer semantics.